docs/build-notifications.rst

```diff
@@ -1,13 +1,13 @@
-Enabling Build Notifications
-============================
+Build Notifications and Webhooks
+================================
 
 .. note::
 
-   Currently we don't send notifications when
-   a :doc:`build from a pull request fails </pull-requests>`.
+   Currently we don't send notifications nor trigger webhooks
```

Comment:
```suggestion
   Currently we don't send notifications or trigger webhooks
```

`nor` isn't really commonly used.

Reply:
Thanks for the real world English wisdom, I really appreciate it 😄